src/components/configuration/sdk-config-docs/ionic.js

```diff
@@ -106,6 +106,25 @@ const framework = docsPrefix => {
             ]
           }
         ]
+      },
+      security: {
+        serviceLogoUrl: '/img/security.svg',
+        serviceName: 'Security',
+        serviceDescription:
+          'Installs a security service which gives developers the ability to control their apps from the Mobile Security Console.',
+        setupText: 'Security SDK setup',
+        docsLink: `${docsPrefix}/security.html`,
+        steps: [
+          {
+            introduction: 'Setting up Mobile Security service SDK',
+            commands: [
+              [
+                'Install AeroGear Sync Cordova plugin',
+                '```cordova plugin add @aerogear/cordova-plugin-aerogear-security```'
```

Comment:
still using cordova plug in for ionic?

Reply:
As far as I know yes, `Push`, `Metrics` and `Sync` are all down as using the cordova plug in for ionic.